src/client/vite.config.ts

```diff
@@ -326,7 +353,7 @@ const setConfig: (env: ConfigEnv) => UserConfigExport = ({ mode }) => {
         },
       ],
     },
-    plugins: isStorybook ? [] : [...plugins],
+    plugins: isStorybook ? [fontfacePreLoad] : [...plugins, fontfacePreLoad],
```

Comment:
bump the logic up outside of `defineConfig()` now, I think
this should be a line like `plugins.push(fontfacePreLoad)` then it is much clearer that it applies in all cases
.. just leaving this
```
    plugins,
```